helm/vertical-pod-autoscaler-app/Chart.lock

```diff
@@ -0,0 +1,6 @@
+dependencies:
```

Comment:
This file is not needed

Reply:
It's automatically generated by the `helm dep up` command. But I can add it to `.gitignore` :)

Reply:
Yes

Reply:
it will be generated by CI

Reply:
So, did you decide to keep this file?

Reply:
File not kept.